integrationtest/mapper/orm-realbackend/src/test/java/org/hibernate/search/integrationtest/mapper/orm/realbackend/schema/management/ElasticsearchSchemaManagerExporterIT.java

```diff
@@ -0,0 +1,151 @@
+/*
+ * Hibernate Search, full-text search for your domain model
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.search.integrationtest.mapper.orm.realbackend.schema.management;
+
+import static org.hibernate.search.util.impl.test.JsonHelper.assertJsonEquals;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import javax.persistence.EntityManagerFactory;
+
+import org.hibernate.search.integrationtest.mapper.orm.realbackend.testsupport.BackendConfigurations;
+import org.hibernate.search.integrationtest.mapper.orm.realbackend.util.Article;
+import org.hibernate.search.integrationtest.mapper.orm.realbackend.util.Book;
+import org.hibernate.search.mapper.orm.Search;
+import org.hibernate.search.util.impl.integrationtest.mapper.orm.OrmSetupHelper;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+public class ElasticsearchSchemaManagerExporterIT {
+
+	@Rule
+	public final TemporaryFolder temporaryFolder = new TemporaryFolder();
+	@Rule
+	public OrmSetupHelper setupHelper = OrmSetupHelper.withSingleBackend( BackendConfigurations.simple() );
+
+	private EntityManagerFactory entityManagerFactory;
+
+	@Test
+	public void elasticsearch() throws IOException {
+		entityManagerFactory = setupHelper.start()
+				.withProperty( "hibernate.search.backend.type", "elasticsearch" )
+				.withProperty( "hibernate.search.backend.version_check.enabled", false )
+				.withProperty( "hibernate.search.backend.version", "8.6" )
+
+				.withProperty( "hibernate.search.backends." + Article.BACKEND_NAME + ".type", "elasticsearch" )
+				.withProperty( "hibernate.search.backends." + Article.BACKEND_NAME + ".version_check.enabled", false )
+				.withProperty( "hibernate.search.backends." + Article.BACKEND_NAME + ".version", "8.6" )
+				.setup( Book.class, Article.class );
+
+		Path directory = temporaryFolder.newFolder().toPath();
+		Search.session( entityManagerFactory.createEntityManager() ).schemaManager().exportSchema( directory );
+
+		String bookIndex = Files.readString(
+				directory.resolve( "backend" ) // as we are using the default backend
+						.resolve( "indexes" )
+						.resolve( Book.class.getName() ) // we use FQN as who knows maybe someone will decide to have same class names in different packages
```

Comment:
That's weird, we should be using the entity name, which is unique within a persistence unit. The entity name should be referred to as "type name" at the backend level, and it's the default for the index name.

Which brings me to this: you're using the index name in the export, right?

Reply:
I think what confused me was that example from the task:
```
backend/indexes/MyEntity/index.json
backends/someNamedBackend/indexes/Foo/index.json
```
seeing that I thought that you wanted to see class names rather than index names 😄. But if it was index name all the time ... well there was a simpler way to get it compared to what I've done initially. It is an index name now.